paasta_tools/cli/cmds/status.py

```diff
@@ -1365,10 +1365,10 @@ def get_versions_table(
     else:
         versions = sorted(versions, key=lambda x: x.create_timestamp, reverse=True)
         config_shas = {v.config_sha for v in versions}
-        if len(config_shas) > 1:
-            show_config_sha = True
-        else:
-            show_config_sha = False
+        show_config_sha = len(config_shas) > 1
+
+        namespaces = {v.namespace for v in versions}
+        show_namespace = len(namespaces) > 1
```

Comment:
do we just want to show namespaces during bounces?? We(CI) use kubectl a lot and now any instance might be on an arbitrary namespace

Reply:
`get_version_table_entry` always shows the namespace regardless of `show_namespace` if you do multiple -v's, e.g. `paasta status -vv`